### ISSUE TITLE
Avoid global css rules and modifying existing css classes

### ISSUE THREAD

**Issue by gashcrumb** [redhat-ipaas/datamapper-ui#19]
With `ViewEncapsulation.None` all of the CSS in the component is applied to the entire page.  This means that changes to existing styles in patternfly or bootstrap will be applied, which will change the entire page layout, not just that of the component.  So for example rules like this can't be used:

```
.row { margin:0px; }

.toolbar-pf { background-color:#f5f5f5; border-bottom:1px solid #d1d1d1; }
.cards-pf { background-color:white; }
.breadcrumb { margin:0px; padding:0px; padding-bottom:10px; padding-left:20px; }
.breadcrumb A { color:#5CBADF; font-weight:bold; }
```

As well as:

```
A:hover { cursor:pointer; }
html, body { height:100%; }
```

Create new CSS classes for these rules and apply them to the elements that need that alignment.

Another option is to try using `ViewEncapsulation.Native` which would be a quick fix, but most likely is only going to work best on Chrome.  Which may be good for the short term, but ultimately we probably can't rely on shadow DOM encapsulation to work cross-browser.

**Comment by gashcrumb** [comment]
Also we'll ensure that the container that we initialize the data-mapper component in is sized appropriately, so you can keep the `html, body` rule in the local `index.html`

**Comment by jbakermd** [comment]
Fixed in commit 3fa8d5b83b54fa6fce49f3bf1ac9533c9a83fd13. I've moved the global styles I need for my testing out into the test index.html, and re-addressed various CSS styles that were accidental global overrides but only apply to Data Mapper UI elements.
